backend/app/services/device_router.py

```
@@ -0,0 +1,115 @@
+from app.services.device_service import device_service
+
+logger = logging.getLogger(__name__)
+
+
+async def route_task_to_device(
+    db: Session,
+    user_id: int,
+    device_id: str,
+    task: TaskResource,
+    subtask: Subtask,
+    team: Kind,
```

Ruff (0.14.14)

Unused function argument: `team`

(ARG001)